animeal/src/Flows/Main/Modules/Home/FeedingPointDetails/Model/FeedingPointDetailsModel.swift

```diff
@@ -129,6 +90,27 @@ final class FeedingPointDetailsModel: FeedingPointDetailsModelProtocol, FeedingP
             }
         }
     }
+
+    private func subscribeForFeedingPointChangeEvents() {
+        self.context.feedingPointsService.feedingPoints
+            .sink { [weak self] result in
+                guard let self else { return }
+                let points = result.uniqueValues
+                let updatedFeeding = points.first {
+                    $0.feedingPoint.id == self.feedingPointId
+                }
+                if let feedingPointModel = updatedFeeding {
+                    self.fullFeedingPoint = feedingPointModel
+                    DispatchQueue.main.async {
```

Comment:
nit: I suggest move logic related to multithreading to view model.

Reply:
moved